src/runner/simple_runner.cpp

```diff
@@ -1,69 +1,14 @@
-#include <wasm/WasmModule.h>
-
 #include <boost/filesystem/operations.hpp>
 #include <boost/filesystem/path.hpp>
+
+#include <faabric/util/bytes.h>
 #include <faabric/util/config.h>
+#include <faabric/util/files.h>
 #include <faabric/util/func.h>
-
+#include <faabric/util/timing.h>
 #include <module_cache/WasmModuleCache.h>
 #include <wamr/WAMRWasmModule.h>
-
-#include <faabric/util/files.h>
```

Comment:
I removed some of the complexity in here, it wasn't quite living up to its name of "simple" runner